app/mailers/user_notifications.rb

```diff
@@ -178,6 +178,15 @@ def account_silenced(user, opts = nil)
     end
   end
 
+  def account_deleted(email, reviewable)
+    post_action_type_id = reviewable.reviewable_scores.first.reviewable_score_type
```

Comment:
Post can be flagged for various reasons like spam, inappropriate etc. First one is provided to the user.